plugins/jabra-gnp/fu-jabra-gnp-device.c

```diff
@@ -603,16 +855,94 @@ fu_jabra_gnp_device_probe(FuDevice *device, GError **error)
 	return TRUE;
 }
 
+static gboolean
+fu_jabra_gnp_device_add_child(FuDevice *device, guint dfu_pid, GError **error)
+{
+	FuJabraGnpDevice *self = FU_JABRA_GNP_DEVICE(device);
+	g_autoptr(FuJabraGnpDevice) child = NULL;
+
+	/* sanity check */
+	if (self->address != FU_JABRA_GNP_ADDRESS_PARENT) {
+		g_set_error(error,
+			    FWUPD_ERROR,
+			    FWUPD_ERROR_NOT_SUPPORTED,
+			    "expected address 0x%x, and got 0x%x",
+			    (guint)FU_JABRA_GNP_ADDRESS_OTA_CHILD,
+			    self->address);
+		return FALSE;
+	}
+
+	child = g_object_new(FU_TYPE_JABRA_GNP_DEVICE, "parent", FU_DEVICE(self), NULL);
```

Comment:
Something seems to be going wrong here. Somehow when setting up a child device, it is NULL which then causes a segfault later on. I can't seem to see the issue though, and it may be related to changes on internals of fwupd. Would you be able to assist, @hughsie ?

Reply:
Do you see this crash on `main` or on `1_9_X`? I'm confused how FU_DEVICE(self) could every be NULL....

Reply:
Perhaps share the BT from gdb catching the segfault for ideas?